crates/biome_js_analyze/src/lib.rs

```diff
@@ -252,11 +252,7 @@ mod tests {
             String::from_utf8(buffer).unwrap()
         }
 
-        const SOURCE: &str = r#"<div
-    class={{
-        "px-2·foo·p-4·bar": ["foo bar p-4"],
-    }}
```

Comment:
I don't think this change is needed. if you have any reasons, could you please explain it.

Reply:
no, i just follow lint docs to run quick test, i forgot to hide it from commit